docs/troubleshooting/troubleshooting-ha.md

```diff
@@ -0,0 +1,51 @@
+# High availability (HA)
+
+High Availability (HA) is designed to automatically restart protected virtual machines in case a host fails. While this helps improve resilience, there are situations where HA may behave unexpectedly, prevent actions from completing, or make recovery more complex.
+
+This page provides guidance on how to understand and resolve common HA-related issues.
```

Comment:
A very common question is "my host rebooted, why did it reboot?"

To make this page really helpful, it would be nice to document how to establish that, by checking whether there are logs that tell that a host self fenced or on the contrary asked another host to reboot (if that can happen).